### PR TITLE
Fix for ticket #8558

### PR DESCRIPTION
**Pull request by may17**

### CODE REVIEW

File: system/modules/core/templates/jquery/j_mediaelement.html5 [hunk 1]
```diff
@@ -9,7 +9,12 @@ $GLOBALS['TL_CSS'][] = 'assets/jquery/mediaelement/'. $GLOBALS['TL_ASSETS']['MED
 <script>
   (function($) {
     $(document).ready(function() {
-      $('video,audio').mediaelementplayer({
+
+      var media_items = $('video').filter(function() {
+        return $(this).parent().prop("tagName") !== "PICTURE";
```
Comment (leofeyer) [inline system/modules/core/templates/jquery/j_mediaelement.html5]: Shouldn't this be `.tagName.toLowerCase() !== 'picture'`? I don't think we need to use `prop()` here.

Comment (may17) [inline system/modules/core/templates/jquery/j_mediaelement.html5]: tagName is no method of jquery and parent returns an jQuery element. So this only works when we use plain javascript but jquery is required so why we shouldn`t use it? Even the filter method is jQuery too :)

See this jsbin:
https://jsbin.com/toxicemoqi/edit?html,js,console,output

Prop returns a string and we could lowercase the value if you like.

Comment (may17) [inline system/modules/core/templates/jquery/j_mediaelement.html5]: And yes we could return the plain javascript element with .parent()[0] or .parent().get(0).

Looks like this: $(this).parent().get(0).tagName.toLowerCase() !== 'picture'

Or in vanilla js:

this.parentElement.tagName.toLowerCase();

Comment (leofeyer) [inline system/modules/core/templates/jquery/j_mediaelement.html5]: Let's just use `prop()`. 😄

**Review by aschempp — APPROVED** [review]
Looks good, though I would keep `$('video,audio')` to not create a new variable. Won't harm to filter the audio elements as well.

File: system/modules/core/templates/jquery/j_mediaelement.html5 (hunk 1 above)
Comment (may17) [inline system/modules/core/templates/jquery/j_mediaelement.html5]: +1 I think thats the cleanest way @aschempp

**Comment by may17** [comment]
@aschempp it looks cleaner for me only to filter first the video items cause thats our goal.

I think you mean something like this:
```javascript
$('video,audio').filter(function() {
    return $(this).parent('picture').length > 0;
}).mediaelementplayer({
```

Both is possible it`s just about personal taste :)

**Comment by aschempp** [comment]
Yeah, though it probably must be reversed, right? The parent *must not* be a picture element 😂

**Comment by aschempp** [comment]
So best probably would be this?

```js
$('video,audio').filter(function() {
    return $(this).parents('picture').length === 0;
}).mediaelementplayer({
```

This will make sure it also works if there is another element around the video, for whatever reason.

**Comment by may17** [comment]
I`ve updated my pull request. I think we`ve found a good solution :)

**Review by aschempp — APPROVED** [review]

**Comment by leofeyer** [comment]
Merged in b6c25522eedc992e30326717d8131aef449ba3ff and updated the XHTML template in 95340c455a010b2f8fbe584e00836e1a7a9b73c8.

**Pull request merged**